server/apps/main/templates/main/my_stories.html

```diff
@@ -0,0 +1,239 @@
+    }
+
+    .navbar {
+      margin-bottom: 20px;
+      padding: 2px !important;
+    }
+
+
+    /* Sticky footer styles
+    -------------------------------------------------- */
+    html {
+      position: relative;
+      min-height: 100%;
+    }
+
+    body {
+      /* Margin bottom by footer height */
+      margin-bottom: 60px;
+    }
+
+    .footer {
+      position: absolute;
+      bottom: 0;
+      width: 100%;
+      /* Set the fixed height of the footer here */
+      height: 60px;
+      background-color: #f5f5f5;
+    }
+
+
+    .fcontainer .text-muted {
+      padding: 0 15px;
+      margin: 20px 0;
+    }
+
+    /* End of application.html inline styles */
+
+    a.navbar-brand {
+      font-weight: 550 !important;
+    }
+
+    a:hover {
+      font-weight: 550 !important;
+      transition: 0.3s;
+    }
+
+    section.stories_page {
+      text-align: center;
+      margin-top: 30px;
+      color: #000;
+    }
+
+    div.story_head p.h3,
+    div.story_body p.h3 {
+      font-weight: 550;
+      font-size: 38px;
+      margin-bottom: 20px;
+    }
+
+    span.p_page {
+      font-weight: 500;
+      font-size: 19px;
+    }
+
+    div.story_head p.h5 {
+      font-size: 22px;
+      margin-top: 40px;
+      font-weight: 550;
+    }
+
+    ul.can_do {
+      list-style: none;
+      max-width: 600px !important;
+      margin: auto;
+    }
+
+    ul.can_do li.do {
+      font-size: 19px;
+      text-align: left !important;
+    }
+
+    div.story_body {
+      margin-top: 60px;
+    }
+
+    .btn {
+      padding: 6px 12px !important;
+      margin-top: 6px !important;
+      font-size: 14px;
+    }
+
+  </style>
+
+  <!-- Navbar -->
+  <div class="container">
+    <nav class="navbar navbar-default">
+      <div class="container-fluid">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar"
+            aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+
+          <!-- Logo -->
+          <a class="navbar-brand" href="{% url 'main:index' %}">
+            AutSPACEs
+          </a>
+        </div>
+
+        <div id="navbar" class="navbar-collapse collapse">
+          <ul class="nav navbar-nav navbar-right">
+            <li {% if section == 'home' %} class="active" {%endif%}>
+              <a href="{% url 'main:index' %}">Home</a>
+            </li>
+            <li {% if section == 'main:list' %} class="active" {%endif%}>
+              <a href="{% url 'main:list' %}">View Experiences</a>
+            </li>
+            <li {% if section == 'main:my_stories' %} class="active" {%endif%}>
+              {% if not request.user.is_authenticated %}
+              <a href="{% url 'main:index' %}">Home</a>
+              {% else% }
```

Comment:
Small pesky bug here. You need to make sure there's a space before the `%` and that the `%` borders the `}`.

```django
{% else %}
```

I wasn't able to see the `My Stories` page otherwise. xD